### ISSUE TITLE
Clipboard: Jerks on the screen when touch copy button on mobile safari

### ISSUE THREAD

**Issue by RomanBorisov** [angular/components#25169]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

This bug only appears on mobile safari if the search bar is at the bottom

### Reproduction

Steps to reproduce:
0. Go to the https://material.angular.io/cdk/clipboard/overview
1. Open safari on iphone device
2. Move the search bar down
3. click to copy button

https://user-images.githubusercontent.com/36147115/175993701-d4dc40ab-5515-4df5-940d-c6f33a639680.mp4

### Expected Behavior

When pressing the button, there should be no jerks.

### Actual Behavior

There are Jerks on the screen when touch copy button

### Environment

- IOS 15.5:
- safari 15.5:

**Comment by crisbeto** [comment]
I just tried it on a device running iOS 15.5 and it didn't happen. I wonder if it's only under a specific set of circumstances.

**Comment by RomanBorisov** [comment]
> I wonder if it's only under a specific set of circumstances.
Yeah, probably. but I tested it on 3 different devices and the result is the same
